modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -144,8 +144,24 @@ properties:
             description: |
               This property works in conjunction with the [enable_shared_bucket_access](#databases-foo_db-enable_shared_bucket_access) property. When **shared bucket access** is enabled, setting this property to true (`"import_docs": true`) indicates that this Sync Gateway node should perform import processing.
 
-              `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
+              Starting in Sync Gateway 2.7, all Sync Gateway nodes can be configured as import nodes. This results in performance benefits as the import process is shared across all Sync Gateway nodes.
+
+              Prior to version 2.7, `import_docs` can only be set to `true` on a single node.
```

Comment:
@priya Is this still true at 2.7? If enable_shared_bucket_access = true doesn't import_docs default to 'true'? Also won't multiple nodes need it set to true in order enable the position of multiple nodes handling the import processing work load? Setting it false enables workload isolation for those nodes excluded from import processing and dedicated to handling remote clients.